### DISCUSSION TITLE
Print on Canon SELPHY CP1500

### DISCUSSION THREAD

**Discussion by DollyCZ** [photobooth-app/photobooth-app#364] [Q&A]
Hi, I'm trying to set up my CP1500 printer to print photos and I can't figure out how to customize the resulting photo. To print from the application I use the command lp -d CP1500 -o landscape -o fit-to-page {filename}

Depending on the settings in the CUPS options with the SELPHY CP1500 HTTP - IPP Everywhere (color) driver, the photo will be printed either with a white border (Postcard setting) or without a border (Postcard.Borderless setting).

I would like to print photos on the Postcard, Borderless settings without borders, but the resulting printed photo is without borders, but it is too enlarged and does not fit on the paper. Does anyone know, please, whether it is possible to use some command or setting to adjust the photo so that the resulting printed photo is complete, but without a frame?

Thank you very much for any help, I am attaching 2 photos of the printout.

![IMG_6433](https://github.com/user-attachments/assets/81a2f799-ed52-457c-8e4b-f8f8d730ff61)
![IMG_6434](https://github.com/user-attachments/assets/289b9901-b8a9-4b62-a508-a44dddbea75f)

**Comment by Striker62** [comment]
Hello, I use a SELPHY CP-1500 with "lp -d SELPHY-CP1500 -o landscape -o fit-to-page {filename}"  same as you and I have no white borders.
My Booth is actually not running but I think you have to check your CUPS settings / Dirver.
 
Or you can try to change the resolution of the camera in the admin section.

**Reply by DollyCZ** [reply]
Thank you very much for the response, I will look forward to it! :)
